src/patterns/data/default/blood_vessel_pattern.tsv

```diff
@@ -0,0 +1,56 @@
+defined_class	label	parent	part_of_vccf	definition	xref_vccf	synonym	synonym_xrefs	connecting_branch_of	tributary_of	vessel_supplies_blood_to	vessel_drains_blood_from	creator									

```

Comment:
The big advantage of DOSDPs over ROBOT is text generation.  This can use optionals too.  If really need to add labels and defs manually, then maybe ROBOT is more appropriate.

Reply:
IMO the main advantage of DOSDPs is that they are a *specification* here, independent of whether you want to use the spec to generate or validate. We are desperately in need of *specifications* of all the patterns in Uberon